model/proxy.py

```diff
@@ -86,8 +86,31 @@ def _required_attributes(self):
             'logging',
         ]
 
+    def _optional_attributes(self):
+        yield from super()._optional_attributes()
+        yield from [
+            'filter_config',
```

Comment:
might it be enough to inline the one (or two) attrs of filter_config?

Reply:
For consistency and (admittedly a _very_ small concern) extensibility I'd rather not.